cairo/tests/utils/strategies.py

```diff
@@ -116,8 +116,13 @@
 MAX_TOUCHED_ACCOUNTS_SIZE = int(os.getenv("HYPOTHESIS_MAX_TOUCHED_ACCOUNTS_SIZE", 10))
 MAX_TUPLE_SIZE = int(os.getenv("HYPOTHESIS_MAX_TUPLE_SIZE", 20))
 
-small_bytes = st.binary(min_size=0, max_size=256)
-code = st.binary(min_size=0, max_size=MAX_CODE_SIZE)
+
+def bounded_bytes_strategy(min_size: int = 0, max_size: int = 2**256 - 1):
+    return st.binary(min_size=min_size, max_size=max_size)
```

Comment:
I don't really think this is useful? `st.binary` has the same default min_size (0) and the max_size is always specified